.github/workflows/docker_release.yml

```diff
@@ -5,36 +5,25 @@ on:
     types: [released, prereleased]
 
 jobs:
-  fetch-latest-tag:
```

Comment:
I made this change because I was using `::set-output`, which is deprecated. Since I was already updating the code, I decided to align the implementation with `docs.yaml` for consistency. In the future, I could move both implementations to a separate file so they can call the same job, but for now, I wanted to ensure they are consistent